dlp/snippets/deid.py

```diff
@@ -18,6 +18,8 @@
 
 import argparse
 
+from typing import List
```

Comment:
duplicated inside the added dlp_deidentify_time_extract as well. remove this one?

Reply:
yes, updated.